<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartySummaryForm</class>
 <widget class="QWidget" name="PartySummaryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>68</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>68</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Party</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="_nameLabel">
        <property name="text">
         <string>Name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fullnameLabel">
        <property name="text">
         <string>&lt;fullname here&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string># Children Affected:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="childrenLabel">
        <property name="text">
         <string>&lt;# children affected&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="spanishCheckbox">
        <property name="text">
         <string>Spanish Required</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
